tests.py

```diff
@@ -426,6 +435,327 @@ def test_plain_numbers(self):
+        self.assertEqual(timeparse.parse('+5 day', as_timedelta=True), relativedelta(seconds = 432000))
+        self.assertEqual(timeparse.parse('-5 day', as_timedelta=True), relativedelta(seconds = -432000))
+
+    def test_timeparse_30(self):
+        """timeparse test case 30."""
+        self.assertEqual(timeparse.parse('5 days', as_timedelta=True), relativedelta(seconds = 432000))
+        self.assertEqual(timeparse.parse('+5 days', as_timedelta=True), relativedelta(seconds = 432000))
+        self.assertEqual(timeparse.parse('-5 days', as_timedelta=True), relativedelta(seconds = -432000))
+
+    def test_timeparse_31(self):
+        """timeparse test case 31."""
+        self.assertEqual(timeparse.parse('5.6 wk', as_timedelta=True), relativedelta(seconds = 3386880))
+        self.assertEqual(timeparse.parse('+5.6 wk', as_timedelta=True), relativedelta(seconds = 3386880))
+        self.assertEqual(timeparse.parse('-5.6 wk', as_timedelta=True), relativedelta(seconds = -3386880))
+
+    def test_timeparse_32(self):
+        """timeparse test case 32."""
+        self.assertEqual(timeparse.parse('5.6 week', as_timedelta=True), relativedelta(seconds = 3386880))
+        self.assertEqual(timeparse.parse('+5.6 week', as_timedelta=True), relativedelta(seconds = 3386880))
+        self.assertEqual(timeparse.parse('-5.6 week', as_timedelta=True), relativedelta(seconds = -3386880))
+
+    def test_timeparse_33(self):
+        """timeparse test case 33."""
+        self.assertEqual(timeparse.parse('5.6 weeks', as_timedelta=True), relativedelta(seconds = 3386880))
+        self.assertEqual(timeparse.parse('+5.6 weeks', as_timedelta=True), relativedelta(seconds = 3386880))
+        self.assertEqual(timeparse.parse('-5.6 weeks', as_timedelta=True), relativedelta(seconds = -3386880))
+
+    def test_milliseconds(self):
+        self.assertEqual(timeparse.parse('3 ms', as_timedelta=True), relativedelta(microseconds= 3000))
+        self.assertEqual(timeparse.parse('3 millis', as_timedelta=True), relativedelta(microseconds = 3000))
+        self.assertEqual(timeparse.parse('3 msecs', as_timedelta=True), relativedelta(microseconds = 3000))
+        self.assertEqual(timeparse.parse('3 milliseconds', as_timedelta=True), relativedelta(microseconds = 3000))
+
+    def test_plain_numbers(self):
+        self.assertEqual(timeparse.parse('10', as_timedelta=True), relativedelta(seconds = 10))
+        self.assertEqual(timeparse.parse('10.1', as_timedelta=True), relativedelta(seconds = 10, microseconds = 100000))
+        self.assertEqual(timeparse.parse('-10', as_timedelta=True), relativedelta(seconds = -10))
+        self.assertEqual(timeparse.parse('-10.1', as_timedelta=True), -relativedelta(seconds = 10, microseconds = 100000))
+
+    def test_combined(self):
+        self.assertEqual(timeparse.parse('1y2mo3w4d5h6m7s8ms', as_timedelta=True), relativedelta(seconds = 38898367, microseconds=8000))
```

Comment:
@2br-2b good work! I think it would be better to refactor this. `TestRelativeDeltaOutput` need just one test method which call some patterned method for each value and check equality of two values:
- value of constant date minus value of `parse(..., as_timedelta=True)`
- value of constant date minus value of `relativedelta(seconds=parse(...))`

What do you think?

Reply:
@2br-2b I've just added support for disabling ``dateutil`` support, because that trick wouldn't work with it.

Reply:
> value of constant date minus value of `relativedelta(seconds=parse(...))`
The only problem is that this won't work with all of the values. For example, `relativedelta(days=365) != relativedelta(years=1)` because some years are leap years. Thus, using relativedelta for parsing months and years won't work. On that note, I think I need to update some of these tests...

I do agree that we could create one function to run the tests; it could take a string and a relativedelta as input and output a boolean.

I'm realizing now; should we have three suites of tests:
1. integer/float output
2. relativedelta output
3. datetime output
?